requirements.txt

```diff
@@ -13,4 +13,5 @@ matplotlib
 pip
 proj
 requests
-xlrd
\ No newline at end of file
+xlrd
+werkzeug
```

Comment:
This is included with Flask itself but it doesn't hurt.

Reply:
Not sure why, but they didn't install with flask for me, had to install both separately. Figured it was worth including since it doesn't hurt.